Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```
@@ -496,6 +496,28 @@
+  id: AsteroidRockTungsten
+  parent: AsteroidRock
+  description: An ore vein rich with wolframite.
+  suffix: Copper
+  components:
+    - type: OreVein
+      oreChance: 1.0
+      currentOre: OreTungsten
+    - type: Sprite
+      layers:
+        - state: rock_asteroid
+        - map: [ "enum.EdgeLayer.South" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)